CHANGELOG.md

```diff
@@ -45,8 +45,6 @@ and is expected to change will be tagged interchangably as "experimental" or
 
 - [ENHANCEMENT] Update Prometheus dependency to v2.26.0. (@rfratto)
 
-- [ENHANCEMENT] Agent running as Windows Service will write logs to the Windows Event log (@mattdurham)
-
```

Comment:
Did you mean for this to get removed or do you consider it part of the general windows support listed above?

Reply:
Yeah, I think since it's related to the windows support feature and it's all in the same release it can just be one entry.